airflow/bin/cli.py

```diff
@@ -1998,6 +2099,14 @@ class CLIFactory(object):
             ('--remove-role',),
             help='Remove user from a role',
             action='store_true'),
+        'user_import': Arg(
+            ("-i", "--import"),
+            metavar="FILEPATH",
+            help="Import users from JSON file"),
```

Comment:
what do you think for putting the example file format here?

Reply:
Sure, I can do that - will update the PR shortly